src/cargo/core/registry.rs

```diff
@@ -134,7 +140,7 @@ impl<'cfg> PackageRegistry<'cfg> {
             Some(&(ref previous, _)) => {
                 if previous.precise() == namespace.precise() {
                     debug!("load/match    {}", namespace);
-                    return Ok(())
+                    return Ok(());
```

Comment:
Perhaps you have rustfmt on auto-run? Could you back out these changes for now? They tend to clutter up the diff unfortunately :(